Documentation/Books/Manual/Views/ArangoSearch/README.md

```diff
@@ -1,10 +1,10 @@
-# Bringing the power of IResearch to ArangoDB
+# ArangoSearch views powered by IResearch in ArangoDB
 
 ## What is ArangoSearch
```

Comment:
How about tweaking the headlines a bit? It currently looks like

![image](https://user-images.githubusercontent.com/7819991/44128039-d1ff0cf2-a040-11e8-83c1-960bcb3d30db.png)

I think we could ditch the "What is ArangoSearch" headline altogether, and promote the h3 headlines to h2 and also scratch the "in ArangoDB" in the h1 headline, as that should be obvious:

![image](https://user-images.githubusercontent.com/7819991/44128082-1b85de96-a041-11e8-9060-f7d9a5a2cbf0.png)